app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/MainNavHost.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Infomaniak SwissTransfer - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.infomaniak.swisstransfer.ui.screen.main
+
+import androidx.compose.foundation.layout.safeDrawingPadding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.toRoute
+import com.infomaniak.swisstransfer.ui.navigation.MainNavigation.*
+
+@Composable
+fun MainNavHost(navController: NavHostController) {
+    NavHost(navController, startDestination = SentDestination, modifier = Modifier.safeDrawingPadding()) {
```

Comment:
Why does the starting destination need to be defined in multiple places? Shouldn't it somehow be defined somewhere together or something?